ui/components/NotificationCenter/index.js

```diff
@@ -226,12 +232,14 @@ const NotificationCountChip = withErrorBoundary(
 
 const Header = withErrorBoundary(({ handleFilter, handleClose }) => {
   const { data } = useGetEventsSummaryQuery();
+  const [selectedSeverity, setSelectedSeverity] = useState('');
```

Comment:
we can get the active filters from the redux store directly

Reply:
Oh right